packages/dds-components/src/components/FileUploader/attr-accept.ts

```diff
@@ -17,22 +17,22 @@
 export function isAccepted(
   file: File,
   acceptedFiles: Array<string> | string | undefined,
-) {
+): boolean {
   if (file && acceptedFiles) {
     const acceptedFilesArray = Array.isArray(acceptedFiles)
       ? acceptedFiles
       : acceptedFiles.split(',');
     const fileName = file.name || '';
     const mimeType = (file.type || '').toLowerCase();
-    const baseMimeType = mimeType.replace(/\/.*$/, '');
+    const baseMimeType = mimeType.match(/^([^/]+)/)?.[1] || '';
```

Comment:
Kanskje enda enklere å bare kjøre en split, så slipper vi regex helt:

```suggestion
    const baseMimeType = mimeType.split('/')[0] || '';
```